RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -216,15 +213,14 @@ void TrackstersMergeProducer::produce(edm::Event &evt, const edm::EventSetup &es
   const auto &seedingTrk = *seedingTrk_h;
   usedSeeds.resize(tracks.size(), false);
 
-  fillTile(tracksterTile, trackstersTRKEM, TracksterIterIndex::TRKEM);
-  fillTile(tracksterTile, trackstersEM, TracksterIterIndex::EM);
-  fillTile(tracksterTile, trackstersTRK, TracksterIterIndex::TRK);
-  fillTile(tracksterTile, trackstersHAD, TracksterIterIndex::HAD);
+  fillTile(tracksterTile, trackstersTRKEM);
+  fillTile(tracksterTile, trackstersEM);
+  fillTile(tracksterTile, trackstersTRK);
+  fillTile(tracksterTile, trackstersHAD);
 
```

Comment:
if you do not want to hard-code the values by hand, as it was, use the `index` from the first trackster in the collection.

Reply:
I put back the hard-coded values as the trackster collection may be empty.